### DISCUSSION TITLE
How to configure Access Request on the Cloud Team version

### DISCUSSION THREAD

**Discussion by sanchezzjf** [gravitational/teleport#30263] [Q&A] [answered]
Hi,

I saw that is possible to configure the Access Request feature in the Team version, but some of the connectors I want to use (such as Teams or email) require some configuration on the host, do I have to create a machine just to be this connector or can I in some way connect to the "host" of the cloud version to configure it there?

**Comment by webvictim** [comment, marked as answer]
Correct. The Enterprise versions of Teleport Cloud do support "hosted" plugins where we run the plugins for you, but the Teleport Team plan requires that you run the plugin yourself and connect it to the Cloud instance.

**Reply by sanchezzjf** [reply]
How can I do it then, I tried to follow the instructions from the github page but without success. It gets confusing, there is a video tutorial or something that might help?
